### DISCUSSION TITLE
Honeywell T9

### DISCUSSION THREAD

**Discussion by tate1435** [d0n4v4nb3ck3r/homebridge-honeywell-home#477] [Q&A]
I’ll probably get slammed for the lame question since I’m super new to this. I have everything setup and do have the thermostat and one sensor in Homebridge and HomeKit. I can’t seem to figure out the fan thing. It will not come on from the Homebridge UI or from the tile in HomeKit. If I don’t show it as a separate tile I get what’s in the screenshot. It will turn on if I toggle the blue button. What am I doing wrong?
![677CBF9E-DCF0-47CE-8F3E-F43A65F9FA36](https://user-images.githubusercontent.com/75556384/115805975-ffbd4680-a3b3-11eb-8668-08e3465eeecf.jpeg)

**Comment by donavanbecker** [comment]
[Fan Mode](https://github.com/donavanbecker/homebridge-honeywell-home/wiki/Fan-Modes)

**Reply by tate1435** [reply]
Thanks for the reply but I'm not sure what to do with the info in the link you provided. I do apologize I'm new to this and learning as I go.
